backend/build_and_push.sh

```diff
@@ -5,6 +5,14 @@
 GIT_SHA=$(git rev-parse --short HEAD)
 ACR_TAG="simplereportacr.azurecr.io/api/simple-report-api-build:$GIT_SHA"
 
+export DOCKER_CLI_EXPERIMENTAL=enabled # to get "manifest inspect"
+
+echo "Trying that inspect plan"
+if docker manifest inspect $ACR_TAG >& /dev/null; then
+    echo "Built image for ${GIT_SHA} already exists in the repository"
+    exit 0
```

Comment:
This is the most interesting part, honestly. 😁

Reply:
Ahhh and we have a small whoops, here, as `sh` does not support `>&`. I think this should be `2>&1`, and this is breaking builds... opening a fix PR now